### ISSUE TITLE
Composer

### ISSUE THREAD

**Issue by ilyabakhlin** [ilyabakhlin/livehelperchat#1]
After performing `composer validate`, the following output is printed:

```
./composer.json is valid, but with a few warnings
See https://getcomposer.org/doc/04-schema.md for details on the schema
# General warnings
- License "GPL-3.0+" is a deprecated SPDX license identifier, use "GPL-3.0-or-later" instead
- License "LGPL-3.0+" is a deprecated SPDX license identifier, use "LGPL-3.0-or-later" instead
- The package "minishlink/web-push" is pointing to a commit-ref, this is bad practice and can cause unforeseen issues.
```

Also, the `composer.json` file needs the following updates:

+ The minimal PHP version required for the project is defined as 5.2.4. However, if you try to perform `composer install` with version 5.6, the following errors are shown:

```
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - brick/math is locked to version 0.9.1 and an update of this package was not requested.
    - brick/math 0.9.1 requires php ^7.1|^8.0 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 2
    - fgrosse/phpasn1 is locked to version 2.1.x-dev and an update of this package was not requested.
    - fgrosse/phpasn1 2.1.x-dev requires php >=7.0.0 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 3
    - guzzlehttp/guzzle is locked to version dev-master and an update of this package was not requested.
    - guzzlehttp/guzzle dev-master requires php ^7.2.5 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 4
    - minishlink/web-push is locked to version dev-master and an update of this package was not requested.
    - minishlink/web-push dev-master requires php >=7.2 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 5
    - psr/http-client is locked to version dev-master and an update of this package was not requested.
    - psr/http-client dev-master requires php ^7.0 || ^8.0 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 6
    - psr/http-factory is locked to version dev-master and an update of this package was not requested.
    - psr/http-factory dev-master requires php >=7.0.0 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 7
    - spomky-labs/base64url is locked to version v2.x-dev and an update of this package was not requested.
    - spomky-labs/base64url v2.x-dev requires php >=7.1 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 8
    - web-token/jwt-core is locked to version v2.2.x-dev and an update of this package was not requested.
    - web-token/jwt-core v2.2.x-dev requires php >=7.2 -> your php version (5.6.40) does not satisfy that requirement.
  Problem 9
    - brick/math 0.9.1 requires php ^7.1|^8.0 -> your php version (5.6.40) does not satisfy that requirement.
    - web-token/jwt-util-ecc v2.2.x-dev requires brick/math ^0.8.17|^0.9 -> satisfiable by brick/math[0.9.1].
    - web-token/jwt-util-ecc is locked to version v2.2.x-dev and an update of this package was not requested.

Failed to install packages for ./lhc_web/composer.json.
```

+ For some reason, the `minimum-stability` configuration is set up to `dev`.

**Comment by ilyabakhlin** [comment]
There's also a problem with Packagist: the repository hasn't been updated since version 2.0 and the name can be improved from `remdex/livehelperchat` to `livehelperchat/livehelperchat`.

**Comment by ilyabakhlin** [comment]
Closed by e1a432c5ef3478ac41ed4351359b66903a83d1f2.
